db.go

```diff
@@ -2672,11 +2675,38 @@ type LSMKeyStatistics struct {
 	BytesRead uint64
```

Comment:
hm right now I am not using this BytesRead field, did you have an idea for if we should use it for rate limiting @jbowens